src/components/atoms/Widget.container.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FunctionComponent } from 'react';
+import { Card } from '@material-ui/core';
+
+interface IProps {
```

Comment:
The empty props pattern is going to repeat itself, and it's a chance for a quick refactor - creating a `NoProps` interface as a utility and use it in such files.
The idea behind it is to show purpose ("I intended to create a component with no props, it's not a placeholder").
This is just an opinion/suggestion, and others would say that it's better to just write an empty interface in every file.
It's your choice :)

Reply:
Personally, I like the empty props pattern, as it allow easy modification in case you want to add/remove props.
Let's discuss about it on our next team meeting :)